codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -335,18 +335,20 @@ update these file paths accordingly.
 
       log.info('uploading ${args.join(' ')}');
       final io.ProcessResult result = processManager.runSync(args);
+      if (result.exitCode < 0 || result.exitCode > 255) {
```

Comment:
Don't we want `if (result.exitCode != 0) {`? For example, exit code of 1 would be not good, right?

Reply:
umm according to process.dart a value in the range of 1 and 255 would be good. An excerpt from dart looks like
```
  /// On Linux and OS X a normal exit code will be a positive value in
  /// the range `[0..255]`. If the process was terminated due to a signal
  /// the exit code will be a negative value in the range `[-255..-1]`,
  /// where the absolute value of the exit code is the signal
  /// number. For example, if a process crashes due to a segmentation
  /// violation the exit code will be -11, as the signal SIGSEGV has the
  /// number 11.
```

Reply:
Right, but if it's not zero, that usually means the process exited because of an error: https://www.gnu.org/software/bash/manual/html_node/Exit-Status.html#:~:text=A%20non%2Dzero%20exit%20status,N%20as%20the%20exit%20status.

Reply:
Ahh thanks for explaining! Update to be nonzero check